src/components/StudentGroup.tsx

```diff
@@ -5,7 +5,7 @@ import styled from '@emotion/styled'
 import 'firebase/firestore'
 import { useFirestore, useFirestoreCollectionData, useUser } from 'reactfire'
 
-const ClassroomDiv = styled.div`
+const StudentGroupDiv = styled.div`
```

Comment:
We don't need to know this is a `div`, let's try for a more descriptive name

Reply:
like StudentGroupOuterContainer?

Reply:
Maybe just `StudentGroupContainer`

Reply:
Okay cool.  Is it okay for me to mark these conversations as resolved, or should I let my reviewer do that?